pkg/gui/gui.go

```diff
@@ -25,6 +20,31 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// lazygitTitle is the icon that gets display when the user focusses
```

Comment:
I'm not sure why this was moved out of the status panel file given that it only pertains to that panel